logger/logger.go

```diff
@@ -25,7 +25,7 @@ type AppLogger struct {
 func GetLogger(logLevel string) AppLogger {
 	var logger log.Logger
 	logger = log.NewLogfmtLogger(os.Stderr)
-	logger = log.With(logger, "ts", log.DefaultTimestampUTC, "caller", log.DefaultCaller)
+	logger = log.With(logger, "ts", log.DefaultTimestampUTC, "caller", log.Caller(6))
```

Comment:
what is magic number 6? please introduce a constant with a descriptive name.

Reply:
set constant DEPTH for `log.Caller(DEPTH)`